IO.Eventuate.Tram.Tests/Messages/MessageTest.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using IO.Eventuate.Tram.Messaging.Producer;
+using IO.Eventuate.Tram.Consumer.Kafka;
+using IO.Eventuate.Tram.Messaging.Common;
+using System.Linq;
+using IO.Eventuate.Tram.Messaging.Producer.Database;
+using IO.Eventuate.Tram.Local.Kafka.Consumer;
+using IO.Eventuate.Tram.Consumer.Common;
+using IO.Eventuate.Tram.Database;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.DependencyInjection;
+using NSubstitute;
+using Microsoft.EntityFrameworkCore;
+
+namespace IO.Eventuate.Tram.Tests
+{
+
+    [TestClass]
+    public class MessageTest
+    {
+        public string uniqueId = System.DateTime.Now.Ticks.ToString();
+        public string subscriberId = "";
+        public string destination = "";
+        public string payload = "";
+        public DatabaseMessageProducer messageProducer;
+        private List<IO.Eventuate.Tram.Messaging.Common.Message> queue = new List<IO.Eventuate.Tram.Messaging.Common.Message>();
```

Comment:
The queue should be `List<IMessage>`

Reply:
I have replaced the above list with _BlockingCollection<IMessage> queue = new BlockingCollection<IMessage>();_ to wait for message to receive.